src/commands/synthetics/utils/internal.ts

```diff
@@ -87,3 +88,97 @@ export const toExecutionRule = (env: string | undefined): ExecutionRule | undefi
 
   return undefined
 }
+
+type validTestOverridesValues = 'boolean' | 'number' | 'string' | 'string[]' | 'ExecutionRule'
+type AccumulatorBaseConfigOverride = Omit<
+  BaseConfigOverride,
+  'basicAuth' | 'headers' | 'cookies' | 'deviceIds' | 'locations' | 'tunnel' | 'variables' | 'retry' // These options will be implemented later in separate PRs, see https://datadoghq.atlassian.net/browse/SYNTH-12971
+> & {
+  'retry.count'?: number
+  'retry.interval'?: number
+}
```

Comment:
I believe `retry` will stay there even after the card is tackled, so let's make this explicit
```suggestion
type AccumulatorBaseConfigOverride = Omit<
  BaseConfigOverride,
  | 'retry'
  // TODO SYNTH-12971: These options will be implemented later in separate PRs
  | 'basicAuth'
  | 'headers'
  | 'cookies'
  | 'deviceIds'
  | 'locations'
  | 'tunnel'
  | 'variables'
> & {
  'retry.count'?: number
  'retry.interval'?: number
}
```